apps/authentication-portal/src/main/resources/org/wso2/carbon/identity/application/authentication/endpoint/i18n/Resources.properties

```diff
@@ -139,6 +139,7 @@ ok=Ok
 mandatory.claims=Mandatory claims
 configuration.error=Configuration Error!
 authentication.mechanism.not.configured=Authentication mechanism not properly configured.
+Error.while.building.the.tenant.qualified.url=Error while building the tenant qualified url.
```

Comment:
This is not thrown just when building the tenant qualified URL right.
Looks to be message needs to be changed

Reply:
Changed the message accordingly and updated the PR.